tests/test_issue_5164_starship_prompt_composition.py

```
@@ -266,7 +267,102 @@ def test_plain_bash_bootstrap_installs_cmux_prompt_command() -> None:
+downstream_prompt_hook
+"""
+    scenarios = {
+        "no transport": {
+            "CMUX_TEST_PATH": "no-transport",
+            "CMUX_SOCKET_PATH": "",
+            "CMUX_TAB_ID": "tab-test",
+            "CMUX_PANEL_ID": "",
+        },
+        "no tab": {
+            "CMUX_TEST_PATH": "no-tab",
+            "CMUX_SOCKET_PATH": "/tmp/cmux-test.sock",
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-test.sock"

(S108)